ostypes_win.jsm

```diff
@@ -164,6 +165,7 @@ var winTypes = function() {
 		{ Pointer: this.PVOID }, //  union { struct { DWORD Offset; DWORD OffsetHigh; }; PVOID Pointer; };
 		{ hEvent: this.HANDLE },
 	]);
+	this.LPOVERLAPPED = this.OVERLAPPED.ptr;
```

Comment:
Also duplicate. `LPOVERLAPPED` is already defined in the section below the section which `OVERLAPPED` was defined in.